RefactoringPracticeI/GameInventory/Inventory/GameInventory.cs

```diff
@@ -1,75 +1,24 @@
-﻿namespace RefactoringExerciseI.Inventory
+﻿using RefactoringExerciseI.Inventory.Handlers;
+
+namespace RefactoringExerciseI.Inventory
 {
     public class GameInventory
     {
         private readonly IList<Item> _items;
+        private readonly ItemUpdateChain _updateChain;
 
         public GameInventory(IList<Item> Items)
         {
             _items = Items;
+            _updateChain = new ItemUpdateChain();
```

Comment:
This dependency could be injected through public accessor and lazy load (relying on an abstraction)
```
private readonly IItemUpdateChain _updateChain;

public IItemUpdateChain ItemUpdateChain {
    get
    {
        if (_updateChain == null)
        {
            _updateChain = new ItemUpdateChain();
        }

        return _updateChain;
    }
    set
    {
        _updateChain = value;
    }
}
```

this way it does not have the responsibility of instantiate its own dependencies, and tomorrow this could have its factory, could be replaced, have a different implementation of ItemUpdateChain, etc.